[2025-01-24T07:02:02.707+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-24T07:02:02.712+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.create_dataset_task manual__2025-01-24T06:54:31.432507+00:00 [queued]>
[2025-01-24T07:02:02.715+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.create_dataset_task manual__2025-01-24T06:54:31.432507+00:00 [queued]>
[2025-01-24T07:02:02.716+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-24T07:02:02.719+0000] {taskinstance.py:2889} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_dataset_task> on 2025-01-24 06:54:31.432507+00:00
[2025-01-24T07:02:02.723+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=91) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-24T07:02:02.723+0000] {standard_task_runner.py:72} INFO - Started process 92 to run task
[2025-01-24T07:02:02.723+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'create_dataset_task', 'manual__2025-01-24T06:54:31.432507+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/GCP Project/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpnnjk065h']
[2025-01-24T07:02:02.725+0000] {standard_task_runner.py:105} INFO - Job 49: Subtask create_dataset_task
[2025-01-24T07:02:02.741+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_gcs_dag.create_dataset_task manual__2025-01-24T06:54:31.432507+00:00 [running]> on host 22ff6bf154fe
[2025-01-24T07:02:02.767+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_gcs_dag' AIRFLOW_CTX_TASK_ID='create_dataset_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-24T06:54:31.432507+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-24T06:54:31.432507+00:00'
[2025-01-24T07:02:02.767+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-24T07:02:02.778+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-01-24T07:02:02.779+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-01-24T07:02:02.779+0000] {bigquery.py:506} INFO - datasetId was not specified in `dataset_reference`. Will use default value trips_data_all.
[2025-01-24T07:02:02.779+0000] {bigquery.py:506} INFO - projectId was not specified in `dataset_reference`. Will use default value de-project-444809.
[2025-01-24T07:02:02.779+0000] {bigquery.py:517} INFO - Creating dataset: trips_data_all in project: de-project-444809 
[2025-01-24T07:02:02.779+0000] {credentials_provider.py:402} INFO - Getting connection using `google.auth.default()` since no explicit credentials are provided.
[2025-01-24T07:02:04.068+0000] {bigquery.py:521} INFO - Dataset created successfully.
[2025-01-24T07:02:04.094+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-24T07:02:04.095+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=create_dataset_task, run_id=manual__2025-01-24T06:54:31.432507+00:00, execution_date=20250124T065431, start_date=20250124T070202, end_date=20250124T070204
[2025-01-24T07:02:04.138+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-24T07:02:04.153+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-24T07:02:04.153+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
